app/src/main/java/com/hfad/mymessenger/CreateMessageActivity.kt

```diff
@@ -39,13 +47,16 @@ class CreateMessageActivity : AppCompatActivity() {
         dataIntent.type = "text/plain"
         //TODO: use putExtra method with the label "Intent.EXTRA_TEXT" to attach the user's message to the intent
         //  and send it to a "Chooser" window for the user to see it with the different apps
+        val message = text.text.toString()
+        dataIntent.putExtra(Intent.EXTRA_TEXT, message)
+        startActivity(dataIntent)
```

Comment:
The comment ask to use a "Chooser", also called a [ShareSheet](https://developer.android.com/training/sharing/send) window.